packages/api/core/src/api/import.ts

```diff
@@ -60,169 +59,212 @@ export default async ({
-      }
-    }
-  };
-
-  await updatePackageScript('start', 'electron-forge start');
-  await updatePackageScript('package', 'electron-forge package');
-  await updatePackageScript('make', 'electron-forge make');
-
-  d('forgified scripts object:', packageJSON.scripts);
-
-  const writeChanges = async () => {
-    await asyncOra('Writing modified package.json file', async () => {
-      await fs.writeJson(path.resolve(dir, 'package.json'), packageJSON, { spaces: 2 });
-    });
-  };
-
-  await writeChanges();
-
-  await asyncOra('Installing dependencies', async () => {
-    d('deleting old dependencies forcefully');
-    await fs.remove(path.resolve(dir, 'node_modules/.bin/electron'));
-    await fs.remove(path.resolve(dir, 'node_modules/.bin/electron.cmd'));
-
-    d('installing dependencies');
-    await installDepList(dir, importDeps);
-
-    d('installing devDependencies');
-    await installDepList(dir, importDevDeps, DepType.DEV);
-
-    d('installing exactDevDependencies');
-    await installDepList(dir, importExactDevDeps, DepType.DEV, DepVersionRestriction.EXACT);
-  });
-
-  await asyncOra('Copying base template Forge configuration', async () => {
-    const pathToTemplateConfig = path.resolve(baseTemplate.templateDir, 'forge.config.js');
-
-    // if there's an existing config.forge object in package.json
-    if (packageJSON?.config?.forge && typeof packageJSON.config.forge === 'object') {
-      d('detected existing Forge config in package.json, merging with base template Forge config');
-      // eslint-disable-next-line @typescript-eslint/no-var-requires
-      const templateConfig = require(path.resolve(baseTemplate.templateDir, 'forge.config.js'));
-      packageJSON = await readRawPackageJson(dir);
-      merge(templateConfig, packageJSON.config.forge); // mutates the templateConfig object
-      await writeChanges();
-      // otherwise, write to forge.config.js
-    } else {
-      d('writing new forge.config.js');
-      await fs.copyFile(pathToTemplateConfig, path.resolve(dir, 'forge.config.js'));
-    }
-  });
-
-  await asyncOra('Fixing .gitignore', async () => {
-    if (await fs.pathExists(path.resolve(dir, '.gitignore'))) {
-      const gitignore = await fs.readFile(path.resolve(dir, '.gitignore'));
-      if (!gitignore.includes(calculatedOutDir)) {
-        await fs.writeFile(path.resolve(dir, '.gitignore'), `${gitignore}\n${calculatedOutDir}/`);
-      }
-    }
-  });
-
-  info(
-    interactive,
-    `
-
-We have attempted to convert your app to be in a format that Electron Forge understands.
-
-Thanks for using ${chalk.green('Electron Forge')}!`
+  const runner = new Listr(
+    [
+      {
+        title: 'Locating importable project',
+        task: async () => {
+          d(`Attempting to import project in: ${dir}`);
+          if (!(await fs.pathExists(dir)) || !(await fs.pathExists(path.resolve(dir, 'package.json')))) {
+            throw new Error(`We couldn't find a project in: ${dir}`);
```

Comment:
```suggestion
            throw new Error(`We couldn't find a project with a package.json file in: ${dir}`);
```

make the error more transparent